### PR TITLE
Don't keywordise HTTP header keys in websocket handler

### PR DESCRIPTION
**Pull request by robert-stuttaford**

### CODE REVIEW

File: src/ring/adapter/jetty9/websocket.clj [hunk 1]
```diff
@@ -87,8 +87,7 @@
      :host (.getHost request)
      :request-method (keyword (.toLowerCase (.getMethod request)))
      :headers (reduce(fn [m [k v]]
-                       (assoc m (keyword
-                                 (string/lower-case k)) (string/join "," v)))
+                       (assoc m (string/lower-case k) (string/join "," v)))
```
Comment (sunng87) [inline src/ring/adapter/jetty9/websocket.clj:90]: Actually there is no need for `lower-case` either

**Comment by robert-stuttaford** [comment]
Fair point, @sunng87 - I removed that too!

**Comment by robert-stuttaford** [comment]
Although -- I did just double check, and all our other Ring headers have lower-case keys - and, ring cookies does expect a lowercase key:

https://github.com/ring-clojure/ring/blob/master/ring-core/src/ring/middleware/cookies.clj#L56

So I put it back :-)

**Comment by sunng87** [comment]
You are right. The SPEC says it's downcased.

**Comment by robert-stuttaford** [comment]
Thank you, @sunng87 ! I greatly appreciate your quick response and effort to produce a new release! 👏

**Pull request merged**